Source/ARTPush.h

```diff
@@ -0,0 +1,70 @@
+//
+//  ARTPush.h
+//  Ably
+//
+//  Created by Ricardo Pereira on 07/02/2017.
+//  Copyright © 2017 Ably. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "ARTTypes.h"
+
+@class ARTRest;
+@class ARTDeviceDetails;
+
+@protocol ARTPushRecipient;
+
+@interface ARTJsonObject : NSDictionary
+@end
+
+@interface ARTDeviceId : NSString
+@end
+
+@interface ARTDeviceToken : NSData
+@end
+
+@interface ARTUpdateToken : NSString
```

Comment:
Yes, that was an initial solution but I changed it to a `typedef` (https://github.com/ably/ably-ios/pull/582/commits/581e76f4d704b6fc69eaeb1ab788d3cccaf15f18#diff-e040b62ac54206ef54da9e6409623314R18). Now it's working fine. 👍